Sources/ScipioKit/Producer/PIF/XCBuildClient.swift

```diff
@@ -4,25 +4,25 @@ import PackageGraph
 import PackageModel
 
 struct XCBuildClient {
-    private let descriptionPackage: DescriptionPackage
     private let buildOptions: BuildOptions
     private let buildProduct: BuildProduct
     private let configuration: BuildConfiguration
+    private let packageLocator: any PackageLocator
     private let fileSystem: any FileSystem
     private let executor: any Executor
 
     init(
-        package: DescriptionPackage,
         buildProduct: BuildProduct,
         buildOptions: BuildOptions,
         configuration: BuildConfiguration,
+        packageLocator: some PackageLocator,
         fileSystem: any FileSystem = localFileSystem,
         executor: any Executor = ProcessExecutor(decoder: StandardOutputDecoder())
```

Comment:
They also should be `some`

Reply:
For `localFileSystem`, that is TSCBasic's global variable and that is typed as `any FileSystem` so cannot be changed. I'll fix `executor` part.

Reply:
Addressed in 67a3cad